scripts/release-crawler/go.mod

```diff
@@ -6,8 +6,4 @@ require github.com/gomarkdown/markdown v0.0.0-20231115200524-a660076da3fd
 
 require github.com/google/go-github/v57 v57.0.0
 
-require (
```

Comment:
Can you explain why you're changing the requirements here?

Reply:
I just run go mod tidy

Reply:
Sorry go version were different for me. I removed the changes